docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,254 @@
+# Using a DB with concord
+
+Sometimes, when you are developing a bot command, you may wish to save some data that will be not deleted across bot restarts. This is where a database comes in. It can save data, and, don't worry, it won't be deleted if you restart your bot.
+
+In this guide, we will be teaching you how to use sqlite3, a local database. It's incredibly fast, easy to use, and simple to debug.
+
+## Opening the DB file
+
+The first step in this guide will be opening the DB file, as it's required for the other sqlite3 functions like reading and writing data.
+
+```c
+#include <sqlite3.h> // Including the sqlite3 header so we can use its functions.
+
+...
+
+sqlite3 *db;
+int rc = sqlite3_open("db.sqlite", &db); // Here we are opening the DB file so other functions (read and save records) can be executed. And also checking the status code of this function, to see if it failed or not to be executed.
+
+if (rc != SQLITE_OK) {  // Checking if something failed while opening the DB file.
+    log_fatal("[SQLITE] Error when opening the DB file. [%s]", sqlite3_errmsg(db)); // As it saw something went wrong, it is going to log a fatal debug message with the error message of what went wrong.
+      
```

Comment:
Same applies here, better move the comment above, and split it into smaller pieces!